bee-queue/arena
# Fix jsoneditor warning

#### Changes Made
This PR adds 5.34.0 jsoneditor files with map file
#### Potential Risks
Warning dissapeared
![Selection_023](https://user-images.githubusercontent.com/14021523/114635350-2a0f5580-9c8a-11eb-93d8-3a28e3298cae.png)


#### Test Plan
1. Go to examples
2. Run any of the examples: npm run start:bull
3. See in your browser console that the warning is not appearing
#### Checklist
